Library/Formula/ctags.rb

```diff
@@ -2,6 +2,7 @@ class Ctags < Formula
   homepage "http://ctags.sourceforge.net/"
   url "https://downloads.sourceforge.net/ctags/ctags-5.8.tar.gz"
   sha1 "482da1ecd182ab39bbdc09f2f02c9fba8cd20030"
+  version "5.8-p1"
```

Comment:
Fixup pushed.  (`revision` is what I was looking for.)

The whole point of publishing a bugfix is so that other people don't run into the bug (I would have saved some frustration had the fix actually made it out of the upstream repository before I ran into it), so I think a revision bump for a bug that's been observed in the wild is justified.